chapters/en/slides/module4/module4_06.md

```diff
@@ -60,62 +70,105 @@ Inside the text file, list the files and folders you would like to ignore, one p
 ```
 
 <br>
-Save the file, and add and commit it with Git. Then try git status again. You should see:
+Save the file, add it to the staging area, and commit the changes with Git. Then try git status again. You should now see that the files are ignored by Git:
 
 ```
+git status
+```
+
+```out
 On branch main
 nothing to commit, working tree clean
 ```
 
 Notes:
-
-Here are the steps you need to take in order to have Git ignore the files and folders that you don't want to be committed to your repo.
+To "hide" folders and files from Git
+we can create a file called `.gitignore` in the root/main directory of our repository
+(the same directory where the `.git` folder is located).
+This slide describes how we can create this file using VS Code,
+and what we need to add to it
+in order to ignore the two files from the previous slide.
 
 ---
 
-# .gitignore tips and tricks
+## .gitignore tips and tricks
 
 <br>
 
 Here are two useful tricks so that you can avoid doing unnecessary work when it comes to `.gitignore`:
 
 <br>
 
-1. Recursive ignoring of filename patterns
+1. Recursively ignoring filename patterns
 
-2. Using `.gitignore_global` to ignore certain patters in every repository
+2. Using `.gitignore_global` to ignore certain patterns in every repository
 
----
+Notes:
+While we could list out every file we want to ignore
+in every project's `.gitignore`,
+this can quickly become tedious to type.
 
-# 1. Recursive ignoring of filename patters
+Instead,
+we can use filename pattern matching and global Git settings
+to simplify this process.
+We will look closer at each of these in the following few slides.
 
-<br>
+---
 
-Append `**/` to the beginning of any file/folder name listed in the `.gitignore` file to have them ignored in subdirectories within the repo as well
+## 1. Recursively ignoring of filename patters
 
 <br>
 
-Here are the steps to follow:
+Wildcards can be used inside `.gitignore` to ignore multiple files with the same name:
 
-- Use a text editor (e.g., VS Code, nano, Jupyter) to create a file called `.gitignore`
+```
+*.log
+```
 
-- Add `**/.ipynb_checkpoints/` to that file and save it
+Prepending `**/` to filenames ignores those files in subdirectories as well:
+
+```
+**/.*log
+```
 
-- `add` and `commit` it with Git
+Starting with a `!` tells git to not ignore a certain directory.
 
-- Type `git status` and see if you no longer see `.ipynb_checkpoints/` as an untracked file
+```
+**/*.log
+!important/.*log
+```
 
 Notes:
+The `.gitignore` file accepts wildcards for matching pattern names,
+similar to what we learned about for the shell in module 2.
```

Comment:
It is good to connect this with module 2.  This section of `.gitignore` was done by Arman, I have just copied here, so could be different to the rest of the chapter.